### DISCUSSION TITLE
parse.py help

### DISCUSSION THREAD

**Discussion by Emend026** [anhourandaquarter/carmina#21] [General]
@anhourandaquarter oops nvm. i forgot to save lol.

**Comment by Emend026** [comment]
@anhourandaquarter *sigh*... I'm so over it. I resolved a merge conflict when I did a pull request of 16-parser but now my pull request is gone? Do you know why that happened? I thought I had to resolve the merge conflict in order for the pull request to go through to everyone. 

sorry for the mess :\

**Reply by infinite0x20** [reply]
I'm actually not sure why the pull request would have closed on its own. It might have been that you closed the pull request by resolving the merge conflict? It's not gone-- it got closed per #22

**Comment by hhchan94** [comment]
I forgot to revise the code on the branch and already pushed it. I tried resetting and reverting, but I'm not really sure how to go about it. So for now, I’m just leaving the main branch alone and copying the changes into the testing branch. Hopefully, I don’t mess anything up!
